x-pack/plugins/security_solution/public/common/components/inspect/modal.tsx

```diff
@@ -101,18 +109,18 @@ export const formatIndexPatternRequested = (indices: string[] = []) => {
 };
 
 export const ModalInspectQuery = ({
+  additionalRequests,
+  additionalResponses,
   closeModal,
-  isShowing = false,
+  inputId,
   request,
   response,
-  additionalRequests,
-  additionalResponses,
   title,
 }: ModalInspectProps) => {
-  if (!isShowing || request == null || response == null) {
```

Comment:
Moved this check into the `inspect/index.test.tsx` where the modal is called from so we don't have to worry about the hooks being called conditionally.